README.md

```diff
@@ -5,22 +5,295 @@
 Realm is a mobile database that runs directly inside phones, tablets or wearables.
 This repository holds the source code for the Realm SDK for Flutter™ and Dart™.
 
-# Alpha
-
 **This project is in the Alpha stage, All API's might change without warning and no guarantees are given about stability. Do not use it in production.**
 
-The previous preview version of Realm Dart can be found on this branch: https://github.com/realm/realm-dart/tree/preview
+## Getting Started
+
+* Import Realm.
+    ```dart
+    import 'package:realm/realm.dart';
+    ```
+
+* Define a data model class `_Car`.
+    ```dart
+    @RealmModel()
+    class _Car {
+      late String make;
+
+      late String model;
+    
+      int? kilometers = 500;
```

Comment:
int? kilometers = 500;
Is this a good sample - nullable int with default value?

Reply:
I don't see why not. Do you think we can improve it?

Reply:
Just wondering isn't it confusing.

Reply:
I don't think so. It demonstrated both default values and nullable properties support